lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -319,26 +319,22 @@ <h4 class="message-title">${_('Your verification will expire soon!')}</h4>
 
         % if course_mode_info['show_upsell'] and not is_course_blocked:
           <div class="message message-upsell has-actions is-shown">
-            <div class="wrapper-tip">
-              <h4 class="message-title">
-                <span class="value">${_("Challenge Yourself!")}</span>
-              </h4>
-              <p class="message-copy">${_("Take this course as an ID-verified student.")}</p>
-            </div>
 
             <div class="wrapper-extended">
-                <p class="message-copy">${_("You can still sign up for an ID verified {cert_name_long} for this course. If you plan to complete the whole course, it is a great way to recognize your achievement. {link_start}Learn more about the verified {cert_name_long}{link_end}.").format(link_start='<a href="{}">'.format(marketing_link('WHAT_IS_VERIFIED_CERT')), link_end="</a>", cert_name_long=cert_name_long)}</p>
-
-              <ul class="actions message-actions">
-                <li class="action-item">
+                <p class="message-copy" align="justify">
+                  <b class="message-copy-bold">
+                    ${_("Pursue a {cert_name_long} to highlight the knowledge and skills you gain in this course.").format(cert_name_long=cert_name_long)}
+                  </b>
+                    ${_("It's official. It's easily shareable. It's a proven motivator to complete the course. {link_start}Learn more about the verified {cert_name_long}{link_end}.").format(link_start='<a href="{}">'.format(marketing_link('WHAT_IS_VERIFIED_CERT')), link_end="</a>", cert_name_long=cert_name_long)}
```

Comment:
NIT: can you also add <br /> just before the learn more link? That way the link appears on a separate line always.